notifiers/matrix.go

```diff
@@ -122,9 +125,11 @@ func (m *matrix) sendMsg(msg, msgType string) error {
 		"body":    msg,
 	}
 
-	nonce := rand.Intn(10000)
-	url := m.Host + matrixUrlPath
-	url = fmt.Sprintf(m.Host+matrixUrlPath, m.Var1, nonce)
+	m.txId += 1
+
+	// create a unique transaction id coming from this matrix client (timestamp + counter)
+	timestamp := time.Now().Nanosecond()
+	url := fmt.Sprintf(m.Host+matrixUrlPath, m.Var1, timestamp, m.txId)
```

Comment:
I think the Nanoseconds Timestamp is sufficient as unique ID since its nearly impossible to get the equal ns Timestamp when ie. sending two Messages :)

Good Work :+1: